Makefile

```diff
@@ -83,7 +83,7 @@ functest-image-push: functest-image-build
 	hack/func-tests-image.sh push
 
 conformance:
-	hack/dockerized "hack/conformance.sh"
+	hack/dockerized "export SKIP_OUTSIDE_CONN_TESTS=${SKIP_OUTSIDE_CONN_TESTS} hack/conformance.sh"
```

Comment:
export VAR is enough maybe ?
```
~ $ AA=5
~ $ env | grep AA
~ $ export AA
~ $ env | grep AA
AA=5
```

Reply:
I saw this is what used [here] for example. (https://github.com/kubevirt/kubevirt/blob/80ebc809c1bcc2ff3a73b50ea96f1a2d9ecdfbe6/Makefile#L31)
What do you think I should do instead?

Reply:
maybe `export SKIP_OUTSIDE_CONN_TESTS` works as well but up to you

Reply:
I prefer to keep it then. Thanks